app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -166,8 +158,14 @@ export default function Donations() {
         </Button>
         <Button
           classes={{
-            root: classes.buttonSecondaryActive,
-            label: classes.buttonSecondaryTextActive,
+            root: classNames(
+              classes.buttonSecondary,
+              classes.buttonSecondaryActive
```

Comment:
confused about the naming. Is active for ":hover"? In that case don't use a separate class, just do `"&:hover": {...}`

Reply:
also you can use `color="secondary"` directly on the `Button` and not worry about css, if that's what you're trying to do

Reply:
Active is meant for when the option is selected (hovered or not). It is the same behaviour as hover, but they are two different selectors.